test/addons-abi/3_callbacks/binding.cc

```diff
@@ -41,8 +41,7 @@ void Init(napi_env env, napi_value exports, napi_value module) {
     { "RunCallback", RunCallback },
     { "RunCallbackWithRecv", RunCallbackWithRecv }
   };
-  status = napi_define_properties(
-    env, exports, sizeof(desc) / sizeof(napi_property_descriptor), desc);
+  status = napi_define_properties(env, exports, 2, desc);
```

Comment:
If you want convenience, use the C++ wrapper. That is what we should recommend for almost anyone using these APIs, as it makes everything much easier.

We have several APIs that take array and count parameters, and in all of them the count parameter is first. Elsewhere I've seen it both ways.